BraintreeDataCollector/src/main/java/com/braintreepayments/api/DataCollectorRequest.kt

```diff
@@ -0,0 +1,5 @@
+package com.braintreepayments.api
+
+data class DataCollectorRequest(
```

Comment:
Also can be TODO but doc strings for this

Reply:
+1 to define what user "consent" means

Reply:
👍 i have a todo item for double checking javadocs and kdocs